README.md

```diff
@@ -169,7 +169,7 @@ Use the npm package manager for Node-based projects. USWDS maintains the [`@uswd
 
 1. Install `Node/npm`. Below is a link to find the install method that coincides with your operating system:
 
-   - Node v12.13.2 (current LTS), [Installation guides](https://nodejs.org/en/download/)
+   - Node v12.13.2 (current LTS), [Installation guides](https://nodejs.org/en/download)
```

Comment:
```suggestion
   - Node LTS (see .nvmrc), [Installation guides](https://nodejs.org/en/download)
```

Reply:
It seems important to note the LTS version to avoid any breaking changes.

Reply:
Maybe it is an edit we can make in #6208 if we are able to update to LTS?